.github/actions/contentful-teardown/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Contentful Teardown
+description: Tears down Contentful Environment if required
+
+inputs:
+  contentful-space-id:
+    description: 'Contentful Space ID'
+    required: true
+  contentful-token:
+    description: 'Contentful access token'
+    required: true
+  contentful-env:
+    description: 'Contentful environment'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Remove Contentful PR Environment
+      if: ${{ inputs.contentful-env != 'Development' }} && ${{ inputs.contentful-env != 'Production' }}
+      shell: bash
+      run: |
+        STATUS_CODE=$(curl --silent \
```

Comment:
i take it there is no way to use the cli for that?

Reply:
There is, but you have to do multiple steps (setup, authenticate, delete and parse all the responses) to do what the API does in one.